wpeverest/everest-forms
# Dev - Aweber integration

### All Submissions:






### Changes proposed in this Pull Request:


Previously, there was no feature to integrate the **Aweber** marketing tool with **Everest Forms**. This PR adds the feature to add **Aweber** subscribers through **Everest Forms**.

Closes # .

### How to test the changes in this Pull Request:

1. Go to **Everest Forms Dashboard > Features> Aweber**
2. Click on **Aweber** toggle button to activate this feature.
3. Click on the **Settings** toggle button and it will redirect to the integration settings page.
4. Generate the Client API key and Client Secret key. (You need to have the **Aweber Developer Account**)
5. Create the **App** and set  redirect URL as "https://yoursuite.com/wp-admin/?evf_aweber_auth=1"
6. Now go to **Everest Forms**
7. To get the access code, as window will open when you need to login to **Aweber** **Note: Here Aweber Account and Aweber developer account is different**
8. Enter the **Client ID** and click on **Get Access Code** button
9. It will generate the **Access Code**.
10. Now, Enter the **Access Code, Secret Key, and Account nickname** on respective fields and click on **Connect to Aweber** button.
11. Now, create the form and perform the integration setups and perform form entry.
12. And check whether the subscriber is added or not in the **Aweber** account.

[PR Dependency](https://github.com/wpeverest/everest-forms-pro/pull/930)

### Types of changes:




### Other information:




### Changelog entry

> Feature - Aweber integration.
